packages/backend/src/domain/eligibility.ts

```diff
@@ -33,24 +33,28 @@ export const filterPrograms = (
   programs: ProgramData[],
   inputData: InputData
 ): Result<ProgramData[], Error> => {
-  const eligibilityResults = programs.map((p) => evaluateRule(p.publicodes, inputData))
+  let filteredPrograms: ProgramData[] = []
+
+  for (const p of programs) {
+    const e = evaluateRule(p.publicodes, inputData)
```

Comment:
nitpick: dans les principes de [cleancode](https://dev.to/mohitsinghchauhan/clean-code-101-variable-naming-conventions-part-1-1c1a), les noms de variables doivent être prononçable et avoir un sens

ici : `e` et `p` on sait pas vraiment ce que c'est ;-)

remplacable par:
`evaluated`/`evaluation`/... et `program` ?

Reply:
Pas convaincu  par les conventional comments. Le fait de mettre "nitpick" ne m'empêche pas d'être piqué dans mon amour propre par ton lien vers un article intitulé "clean code 101" ! Je plaide donc pour ma défense que c'est une [pratique encouragée en Golang pour des variables avec un scope réduit](https://github.com/golang/go/wiki/CodeReviewComment#variable-names). 

Voilà, maintenant que j'ai sauvé les apparences et comme ce n'est pas du golang, je modifie :wink: